routes/app/signin.js

```diff
@@ -24,13 +24,13 @@ router.post('/signin', async function(req, res) {
         message: 'success'
       });
     } else {
-      res.status(401); // send unauthorized
+      throw new Error('Password is incorrect.'); 
     }
   } catch (error) {
     Logger.error(error);
-    res.status(500).json({
-      error: error.message
-    })
+    res.json({
```

Comment:
We should still set the status here to be `.status(401)` because the user would be unauthorized